<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SslClientSocketConfigWidget</class>
 <widget class="QWidget" name="SslClientSocketConfigWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>483</width>
    <height>456</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SSL Client Socket Configuration Widget</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="m_hostLabel">
       <property name="text">
        <string>Remote host:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="m_hostLineEdit"/>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="m_portLabel">
       <property name="text">
        <string>Remotel Port:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="m_portSpinBox"/>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="m_caDirLabel">
       <property name="toolTip">
        <string/>
       </property>
       <property name="text">
        <string>CA Directory:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="m_caDirLineEdit"/>
     </item>
     <item>
      <widget class="QToolButton" name="m_caDirToolButton">
       <property name="text">
        <string>...</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="m_caDirFormatComboBox"/>
     </item>
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_10">
     <item>
      <widget class="QLabel" name="m_caFileLabel">
       <property name="toolTip">
        <string/>
       </property>
       <property name="text">
        <string>CA File(s):</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="m_caFileLineEdit"/>
     </item>
     <item>
      <widget class="QToolButton" name="m_caFileToolButton">
       <property name="text">
        <string>...</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="m_caFileFormatComboBox"/>
     </item>
     <item>
      <spacer name="horizontalSpacer_10">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QLabel" name="m_verifyHostLabel">
       <property name="text">
        <string>Verify Host:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="m_verifyHostComboBox"/>
     </item>
     <item>
      <spacer name="horizontalSpacer_4">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_9">
     <item>
      <widget class="QLabel" name="m_verifyNameLabel">
       <property name="text">
        <string>Verify Name:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="m_verifyNameLineEdit"/>
     </item>
     <item>
      <spacer name="horizontalSpacer_9">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_6">
     <item>
      <widget class="QLabel" name="m_protocolLabel">
       <property name="text">
        <string>Protocol:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="m_protocolComboBox"/>
     </item>
     <item>
      <spacer name="horizontalSpacer_6">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <widget class="QLabel" name="m_certLabel">
       <property name="toolTip">
        <string>comma separated</string>
       </property>
       <property name="text">
        <string>Local Cerificate File:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="m_certLineEdit"/>
     </item>
     <item>
      <widget class="QToolButton" name="m_certToolButton">
       <property name="text">
        <string>...</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="m_certFormatComboBox"/>
     </item>
     <item>
      <spacer name="horizontalSpacer_5">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_7">
     <item>
      <widget class="QLabel" name="m_privKeyLabel">
       <property name="text">
        <string>Private Key File:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="m_privKeyLineEdit"/>
     </item>
     <item>
      <widget class="QToolButton" name="m_privKeyToolButton">
       <property name="text">
        <string>...</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="m_privKeyAlgComboBox"/>
     </item>
     <item>
      <widget class="QComboBox" name="m_privKeyFormatComboBox"/>
     </item>
     <item>
      <spacer name="horizontalSpacer_7">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_8">
     <item>
      <widget class="QLabel" name="m_privKeyPassLabel">
       <property name="text">
        <string>Private Key Password:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="m_privKeyPassLineEdit">
       <property name="echoMode">
        <enum>QLineEdit::Password</enum>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_8">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
